web/src/routes/(user)/archive/+page.svelte

```diff
@@ -193,10 +192,7 @@
 	<svelte:fragment slot="header">
 		{#if isMultiSelectionMode}
 			<ControlAppBar
-				on:close-button-click={() => {
-					assetInteractionStore.clearMultiselect();
```

Comment:
`clearMultiSelectAssetAssetHandler` clear the local state while `assetInteractionStore.clearMultiselect();` clear the global state

Reply:
I could restore a call of `assetInteractionStore.clearMultiselect` but I'm pretty sure that its removal is correct. `GalleryViewer` doesn't use `assetInteractionStore`. It just binds to `selectedAssets` container. Need you review again